docs/accounting/shell.nix

```diff
@@ -0,0 +1,30 @@
+with (import <nixpkgs> {});
+
+let pkgs = import ../../pkgs.nix; in
```

Comment:
If you want to do it this way, then you don't want the `with (import <nixpkgs> {});` as this will import the `nixpkgs` from your local path.  `with (import ../../pkgs.nix);` should work instead of these two lines and then you can remove `pkgs` from the front of `haskellPackages.lhs2tex`.